src/types/model.ts

```diff
@@ -251,30 +253,30 @@ export const getAccessManagementSummary = (accessManagement: string): AccessMana
 }
 
 export interface DataUse {
-  generalUse: boolean
-  hmbResearch: boolean
-  diseaseRestrictions: string[]
-  populationOriginsAncestry: boolean
-  methodsResearch: boolean
-  nonProfitUse: boolean
-  other: string
-  secondaryOther: string
-  ethicsApprovalRequired: boolean
-  collaboratorRequired: boolean
-  geographicalRestrictions: string
-  geneticStudiesOnly: boolean
-  publicationResults: boolean
-  publicationMoratorium: string
-  controls: boolean
-  gender: string
-  pediatric: boolean
-  population: boolean
-  illegalBehavior: boolean
-  sexualDiseases: boolean
-  stigmatizeDiseases: boolean
-  vulnerablePopulations: boolean
-  psychologicalTraits: boolean
-  notHealth: boolean
+  generalUse?: boolean
```

Comment:
Should all of these be optional?

Reply:
Individually, yes, each field can be optional. Holistically, at least one of `generalUse`, `hmbResearch`, or `diseaseRestrictions` must be present to ensure a primary data use. I'll look into seeing if TS can handle that.